### PR TITLE
docs: Add white paper on the correction term in transport jacobian

### PR DESCRIPTION
**Pull request by beomki-yeo**
@felix-russo @paulgessinger Could you guys transfer the ownership?

Abstract:
```
In the track extension model of ACTS, the covariance of a track is updated
for every surface intersection by applying a jacobian matrix, the composition of
the coordinate transformation and free space transport. In the derivation of the
transport jacobian, it is necessary to consider the path difference coming from the
track displacement, which appears as a direction variation term with respect to the
path length.
The motivation of this white paper is that the author failed in finding any refer-
ence explaining this clearly. (Please let the author know if anyone finds it) There
are bunch of references which calculate this term with an ideal helix model, but
it is not very useful with the runge-kutta based track extension model. There-
fore, in this note, we are deriving the term thoroughly and demonstrate how it is
implemented in the source code.
```

### CODE REVIEW

File: docs/white_papers.toml [hunk 1]
```diff
@@ -16,3 +16,14 @@ pdf_url = "https://github.com/felix-russo/gaussian-track-densities/releases/down
...
+
+[[white_papers]]
+repository = "https://github.com/beomki-yeo/path-correction-term-from-momentum-variation"
+slug = "correction-for-transport-jacobian"
+pdf_url = "https://github.com/beomki-yeo/path-correction-term-from-momentum-variation/releases/download/v1.0/main.pdf"
+
+[white_papers.metadata]
+authors = [ "Beomki Yeo",]
+title = "A correction term for transport jacobian from the direction variation"
+description = "This note mathematically derives the correction term for the transport jacobian and explains how it is implemented in ACTS source code"
```
Comment (felix-russo) [inline docs/white_papers.toml:28]: I think the abstract is not rendered correctly... Did you put it in by hand? Or did you use `white_papers.py` to generate the metadata?

File: docs/white_papers.toml [hunk 2]
```diff
@@ -16,3 +16,14 @@ pdf_url = "https://github.com/felix-russo/gaussian-track-densities/releases/down
...
 description = "Given the impact parameters $(d_0, z_0, t_0)$ of a track in Perigee parametrization, one can model the probability of the particle passing exactly through a point $(d, z, t)$ using a multivariate Gaussian distribution. In this white paper, we derive the maximum and the width of such a distribution for $d = 0$. This is useful in vertex seed finding, where we only consider the track density along the beam axis. We use the analytical results from this white paper in the unit test of the Acts module AdaptiveGridTrackDensity."
+
+[[white_papers]]
+repository = "https://github.com/beomki-yeo/path-correction-term-from-momentum-variation"
+slug = "correction-for-transport-jacobian"
+pdf_url = "https://github.com/beomki-yeo/path-correction-term-from-momentum-variation/releases/download/v1.0/main.pdf"
+
+[white_papers.metadata]
+authors = [ "Beomki Yeo",]
+title = "A correction term for transport jacobian from the direction variation"
```
Comment (felix-russo) [inline docs/white_papers.toml:27]: nitpick: The title should be capitalized correctly, see https://capitalizemytitle.com/ for example :)

**Review by felix-russo — COMMENTED** [review]
The docs build fails because the screenshot of your paper is missing:
image

Did you forget to commit it? Or did you maybe not execute `white_papers.py`?

**Comment by paulgessinger** [comment]
This is amazing @beomki-yeo! Thanks!

**Comment by paulgessinger** [comment]
This is what this currently looks like rendered on the docs page: https://acts--2643.org.readthedocs.build/en/2643/white_papers/correction-for-transport-jacobian.html

File: docs/white_papers.toml (hunk 1 above)
Comment (beomki-yeo) [reply docs/white_papers.toml:28]: I put it by hand...

Comment (beomki-yeo) [reply docs/white_papers.toml:28]: ok I will try to do that more properly

**Comment by beomki-yeo** [comment]
My personal feedback: installing all the reuquirements is kinda difficult with old python version :/

File: docs/white_papers.toml (hunk 2 above)
Comment (beomki-yeo) [reply docs/white_papers.toml:27]: Which style are you exactly following? Not all journal publications have their titles in captilized style

File: Core/src/Propagator/detail/JacobianEngine.cpp [hunk 3]
```diff
@@ -138,6 +138,7 @@ BoundMatrix boundToBoundTransportJacobian(
   // Calculate the jacobian from free to bound at the final surface
   FreeToBoundMatrix freeToBoundJacobian =
       surface.freeToBoundJacobian(geoContext, freeParameters);
+  // https://acts.readthedocs.io/en/latest/white_papers/path-correction-term-from-momentum-variation.html
```
[the remark this answers is not in the document]
Comment (beomki-yeo) [reply Core/src/Propagator/detail/JacobianEngine.cpp]: Fixed

File: docs/white_papers.toml (hunk 2 above)
Comment (beomki-yeo) [reply docs/white_papers.toml]: Capitalization is done anyway

**Comment by paulgessinger** [comment]
@beomki-yeo I see. I'm thinking of a way to automatize the update process so it can just run in the CI. The other option @krasznaa suggested yesterday is a docker container with the requirements of course.

**Comment by beomki-yeo** [comment]
Thanks. Either way would be great

**Review by paulgessinger — DISMISSED** [review]

File: docs/white_papers.toml [hunk 4]
```diff
@@ -15,4 +15,13 @@ pdf_url = "https://github.com/felix-russo/gaussian-track-densities/releases/down
 [white_papers.metadata]
 authors = [ "Felix Russo",]
 title = "Gaussian Track Densities"
-description = "Given the impact parameters $(d_0, z_0, t_0)$ of a track in Perigee parametrization, one can model the probability of the particle passing exactly through a point $(d, z, t)$ using a multivariate Gaussian distribution. In this white paper, we derive the maximum and the width of such a distribution for $d = 0$. This is useful in vertex seed finding, where we only consider the track density along the beam axis. We use the analytical results from this white paper in the unit test of the ACTS module AdaptiveGridTrackDensity."
+description = "Given the impact parameters $(d_0, z_0, t_0)$ of a track in Perigee parametrization, one can model the probability of the particle passing exactly through a point $(d, z, t)$ using a multivariate Gaussian distribution. In this white paper, we derive the maximum and the width of such a distribution for $d = 0$. This is useful in vertex seed finding, where we only consider the track density along the beam axis. We use the analytical results from this white paper in the unit test of the Acts module AdaptiveGridTrackDensity."
```
Comment (felix-russo) [inline docs/white_papers.toml:18]: Is this wanted that the metadata for the previous white papers is updated @paulgessinger?

Comment (felix-russo) [reply docs/white_papers.toml:18]: Also, I didn't create a new version since the Gaussian Track Density white paper was created, so this is a bit curious

Comment (paulgessinger) [reply docs/white_papers.toml:18]: Well, *ACTS* is changed to *Acts* in the abstract.

Comment (paulgessinger) [reply docs/white_papers.toml:18]: So the change is already in the latest version that you made. So my best guess is the file was not regenerated after you made the release?

Comment (felix-russo) [reply docs/white_papers.toml:18]: That is possible!

**Review by paulgessinger — APPROVED** [review]

**Comment by acts-project-service** [comment]
## :red_circle: Athena integration test results [cf77a534090c26086660d20932a82569e7578767]
### :red_circle: Some tests have failed!
Please investigate the [pipeline](https://gitlab.cern.ch/acts/acts-athena-ci/-/pipelines/6461979)!

|status|job|
|------|---|
|:red_circle: | [run_workflow_tests_run2_data](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/33847124) |
|:green_circle: | [run_unit_tests](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/33846252) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsBenchmarkWithSpot.sh 8 100](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/33846251) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsWorkflow.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/33846250) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsValidateAmbiguityResolution.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/33846249) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsValidateTracks.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/33846248) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsValidateActsCoreSpacePoints.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/33846247) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsValidateActsSpacePoints.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/33846246) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsValidateSeeds.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/33846245) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsValidateOrthogonalSeeds.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/33846244) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsValidateClusters.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/33846243) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsPersistifyEDM.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/33846242) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsKfRefitting.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/33846241) |
|:green_circle: | [run_ci_tests: python3 ../athena/Tracking/Acts/ActsGeometry/test/ActsExtrapolationAlgTest.py](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/33846240) |
|:green_circle: | [run_ci_tests: python3 ../athena/Tracking/Acts/ActsGeometry/test/ActsITkTest.py](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/33846239) |
|:green_circle: | [run_workflow_tests_run4_mc](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/33846238) |
|:red_circle: | [run_workflow_tests_run3_mc](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/33846236) |
|:red_circle: | [run_workflow_tests_run3_data](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/33846235) |

**Pull request merged**